bom/pom.xml

```diff
@@ -326,9 +326,9 @@ THE SOFTWARE.
 
       <!--XStream-->
       <dependency>
-        <groupId>org.jvnet.hudson</groupId>
+        <groupId>com.thoughtworks.xstream</groupId>
```

Comment:
If merged, we should also archive both https://github.com/jenkinsci/xstream and https://github.com/jenkinsci/xstream-fork.